src/models/game.js

```diff
@@ -12,17 +12,18 @@ export default class Game {
         this.board = new Board();
     }
 
```

Comment:
get gameHasEnded() {
   return this.board.gameState == GameState.Won || this.board.gameState == GameState.Draw
}